Sources/Sentry/include/SentrySpan.h

```diff
@@ -5,15 +5,46 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class SentryTracer;
+@class SentryTracer, SentryId, SentrySpanId;
 
 @interface SentrySpan : NSObject <SentrySpan, SentrySerializable>
 SENTRY_NO_INIT
 
 /**
- * The context information of the span.
+ * Determines which trace the Span belongs to.
  */
-@property (nonatomic, readonly) SentrySpanContext *context;
+@property (nonatomic) SentryId *traceId;
+
+/**
```

Comment:
`m`: As this class is not public, we can skip the docs. We have them in the SnetrySpanProtocol. The same applies for SentryTracer.h

Reply:
You're right. This docs were in the SpanContext and I just copied everything. 
I dont think is worth to remove it now.

Reply:
I would disagree in general @philipphofmann, it would be nice to have anything in a header documented whether it is public facing or not, although a lot of these are superfluous so could be removed or improved, like:
```
/**
 * Span id.
 */
 @property (nonatomic) SentrySpanId *spanId;
```

Reply:
I was referring to docs that are duplicate. We already have them in the `SentrySpanProtocol`. Other languages have a `@inheritdoc` options or something similar. So you don't write the same docs in two places.

If we complete https://github.com/getsentry/sentry-cocoa/issues/2278 with DocC we could use 
`--enable-inherited-docs`.